app/src/main/res/values-b+nb/strings.xml

```diff
@@ -3,30 +3,30 @@
     <string name="app_name_lite">Soul EV Spy Lite</string>
 
     <!-- Navigation Drawer -->
-    <string name="action_bluetooth">Koble til OBD-II dongle</string>
+    <string name="action_bluetooth">Koble til med Bluetooth</string>
```

Comment:
Mobilen kan jo også forbinde til bilen med bluetooth (det er der brugere der har taget fejl af), så jeg ser meget gerne at OBD indgår i denne tekst.

Reply:
Ta en kikk på mine siste to commits, ser det greit ut nå?